Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3588,9 +3591,22 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
 
 id UnifiedPDFPlugin::accessibilityObject() const
 {
-#if PLATFORM(MAC)
     return m_accessibilityDocumentObject.get();
-#endif
+}
+
+id UnifiedPDFPlugin::accessibilityHitTestInPage(WebCore::FloatPoint point)
+{
+    RefPtr pageRoot = this->page();
+    if (!pageRoot)
+        return nil;
+
+    auto [page, pointInPage] = rootViewToPage(pageRoot->chrome().screenToRootView(WebCore::IntPoint(point)));
+
+    if (!page)
```

Comment:
Small style thing, but wondering if the new line separating `if (!page)` and `auto [page, pointInPage]` is necessary?